include/cli_utils.h

```diff
@@ -44,15 +42,22 @@ Status unified_sites_of_yaml(const YAML::Node& yaml,
                              const std::vector<std::pair<std::string,size_t> > &contigs,
                              std::vector<unified_site> &sites);
 
+Status merge_all(const std::vector<discovered_alleles> &valleles,
+                 discovered_alleles &dsals);
+
 // Merge a bunch of discovered-allele files, all in YAML format.
 //
-// Note: each file could have a different set of containing ranges, and we want to merge
-// alleles in each range separately.
 Status merge_discovered_allele_files(std::shared_ptr<spdlog::logger> logger,
                                      const std::vector<std::string> &filenames,
                                      std::vector<std::pair<std::string,size_t>> &contigs,
-                                     std::vector<range> &ranges,
-                                     std::vector<discovered_alleles> &valleles);
+                                     discovered_alleles &dsals);
+
+
+// Find which range contains [pos].
```

Comment:
Sorry, there is something here that I don't understand. Since the ranges parameter has been removed, which ranges are we referring to?